ZionEscape/MessageBox.h

```diff
@@ -0,0 +1,76 @@
+#pragma once
+
+#ifndef _MESSAGEBOX_H_
+#define _MESSAGEBOX_H_
+
+#include "BitmapManager.h"
+
+ref class Messagebox {
+  bool activated;
+  BitmapManager^ bmpManager;
+  int counter, len, order, nextMessage;
+  String^ txt;
+public:
+  Messagebox() {
+    this->bmpManager = BitmapManager::GetInstance();
+    this->counter = len = order = nextMessage = 0;
+  }
+  ~Messagebox() {
+    delete this->bmpManager;
+    delete this->txt;
+  }
+  void Draw(Graphics^ g) {
+    //Select the image and the position
+    g->DrawImage(bmpManager->GetImage("assets\\sprites\\misc\\messagebox.png"), Point(101,230));
+  }
+
+  void SetActivated(bool value) {
+    this->activated = value;
+  }
+
+  void SetCounter(int value) {
+    this->counter += value;
+  }
+
+  void SetNextMessage(int value) {
+    this->nextMessage += value;
+  }
+
+  void SetOrder(int value) {
+    this->order += value;
+  }
+
+  void SetLen(int value) {
```

Comment:
Trata de poner el nombre completo, si no se entiende qué es realmente.

```suggestion
  void SetLength(int value) {
```